docs/topics/wasm/wasm-js-interop.md

```diff
@@ -359,7 +359,57 @@ external fun <T : JsAny> processData(data: JsArray<T>): T
 
 ## Exception handling
 
-The Kotlin `try-catch` expression can't catch JavaScript exceptions.
+Since Kotlin 2.0.0 the Kotlin `try-catch` expression can catch JavaScript exceptions.
```

Comment:
I recently learned from Andrey we try to avoid these phrases as our docs must reflect the last state of the language. If this was a status in 2.0.0 that then changed in 2.1.0 I suggest describing it in such a manner it reflects the latest status.

Reply:
ok - I restructured the two paragraphs